java/src/test/java/ai/rapids/cudf/ast/CompiledExpressionTest.java

```diff
@@ -42,15 +42,15 @@
   public void testColumnReferenceTransform() {
     try (Table t = new Table.TestBuilder().column(5, 4, 3, 2, 1).column(6, 7, 8, null, 10).build()) {
       // use an implicit table reference
-      UnaryExpression expr = new UnaryExpression(UnaryOperator.IDENTITY,
+      UnaryOperation expr = new UnaryOperation(UnaryOperator.IDENTITY,
```

Comment:
This shouldn't be an identity operation anymore, we can use the column reference directly.  This comment applies to all the places where we're using a unary identity operation as a hack around the old restriction that column references and literals couldn't be used directly as expressions.

Reply:
Done. In the process of getting this to work I had to modify `compile_serialized_ast` to support literals and column references, and now that they cover all the same types I modified it to just call `compile_expression` directly. Let me know if that causes any problems that I'm not anticipating.